backend/models/FeatureFlag.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const schema = mongoose.Schema;
+
+const FeatureFlagSchema = schema({
+    name: {
+        type: String,
+        require: true
+    },
+    roleUsers: {
+        type: String,
+        require: true
+    },
```

Comment:
It should be of type array of strings and I think just calling the field `roles` would suffice. What do you think?